src/main.js

```
@@ -1,30 +1,199 @@
 
-function createWindow() {
-    const win = new BrowserWindow({
-        width: 1600,
-        height: 1000,
-        webPreferences: {
-            // skipcq: JS-S1019
-            nodeIntegration: true,
-            // skipcq: JS-S1020
-            contextIsolation: false
+const log = require('electron-log')
+log.transports.file.level = 'info';
```

Biome (1.9.4)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)